ansible_rulebook/collection.py

```diff
@@ -134,6 +137,24 @@ def load_rulebook(collection, rulebook):
         return yaml.safe_load(f.read())
 
 
+def has_action(collection, action):
```

Comment:
@benthomasson if we can take in an additional parameter additional_dirs, here which is the directory names we can search for the files in the collection location and an additional directory

```
    def has_action(collection, action, additional_dirs = []):
       dirs_to_search = additional_dirs
       dirs_to_search.append(EDA_ACTION_PATHS)
       return has_object(
        collection,
        action,
        dirs_to_search,
        ".py",
    )
           
```